app/main.py

```diff
@@ -1,8 +1,11 @@
 from fastapi import FastAPI
-from schema import (Score)
+from app.schema import (Score)
+import boto3
 
-api_version = 1
+api_version = "1.0.0"
+dynamodb = boto3.resource('dynamodb')
 app = FastAPI()
+Games = dynamodb.Table('GamesTest')
```

Comment:
Typically, python users would expect only classes to be upper-cased. If you want to denote that the variable is a table, you could either rely on mypy/IDE:

```
games: dynamodb.Table = dynamodb.Table('GamesTest')
```

or maybe add a suffix to the variable name: `games_tbl = dynamodb.Table('GamesTest')` – I think you want to err on the side of fewer/no suffixes.